habitat_sim/simulator.py

```diff
@@ -106,20 +106,20 @@ def __attrs_post_init__(self) -> None:
         self.__set_from_config(self.config)
 
     def close(self) -> None:
-        for sensor in self._sensors.values():
-            sensor.close()
-            del sensor
+        for agent_sensorsuite in self.__sensors:
+            for sensor in agent_sensorsuite.values():
+                sensor.close()
+                del sensor
 
-        self._sensors = {}
+        self.__sensors = []
```

Comment:
Why was this changed to `__`?

Reply:
For backwards compatibility. It's not a public API, but we have a lot of calls into. I emulate the old behavior with an '@property' getter.

Reply:
Hmm, well, we should make a public API for getting the sensor objects.

Reply:
Yeah, that can be a separate PR though.